Updating Recommendations docs based on feedback (#6691)
magento/devdocs: src/recommendations/troubleshooting.md

---
group: product-recommendations
title: Troubleshoot Recommendations
---

If you have configured the `product-recommendations` module correctly, but you are not seeing any recommendations, try the following:

> -  It is possible that the module has not had enough time to collect behavioral data. Allow the system to run for 24 hours so it can start collecting data.

Reviewer: One thing we can recommend here is to deploy a MLT rec type first (requires no behavioral data)

-  If you are not seeing the recommendations that you configured, it is possible there is not yet sufficient data to build recommendations for the user.

> -  Ensure the SaaS Environment ID or API Key are valid. If you get an error after specifying your SaaS Environment ID or your API key during the product recommendations initialization, check to make sure you have entered the [SaaS Environment ID]({{ page.baseurl }}/recommendations/configure.html#envid) and [API key]({{ page.baseurl }}/recommendations/configure.html#apikeys) correctly.

Reviewer: need to add the same note here about mageID and API keys having to match

> -  Check the logs. Make sure there are no errors associated with the above modules.

Reviewer 2: Add a link to a topic that discusses how to check the logs. Are there new or specific logs the reader needs to consult?

Author: Added link, but no new logs.

> -  Confirm the indexers and cron jobs are running.

Reviewer 2: Add a iink

-  Refresh Configuration cache. Go to **System** > **Tools** > **Cache Management**, and clear the configuration cache.

> -  Confirm the `Product Feed` indexer is set to `Update by Schedule`.

Reviewer 2: Maybe combine with the "Confirm indexers are running" list item?

-  Confirm there is data in the `catalog_data_exporter_products` database table.

### Events

> Refer to the [following]({{ page.baseurl }}/recommendations/verify.html) page for help in verifying behavioral events are being sent to Magento.

Reviewer 2: ```suggestion
[Recommendation Events]({{ page.baseurl }}/recommendations/verify.html) describes the behavioral events that are sent to Magento.
```
